recipes/numpydoc/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.6" %}
+
+package:
+  name: numpydoc
+  version: {{ version }}
+
+source:
+  fn: numpydoc-{{ version }}.tar.gz
+  url: https://github.com/numpy/numpydoc/archive/v{{ version }}.tar.gz
```

Comment:
This is generally not the right thing to do. This link points to a tarball automatically created by GitHub, which is not the same as what one gets from ``python setup.py sdist`` and uploads to PyPI. For ``numpy`` and ``scipy`` we tend to fix that by uploading tarballs to GitHub Releases, but not for ``numpydoc``. So link should be to PyPI.